web-app-router/components/Radio/index.tsx

```diff
@@ -1,7 +1,47 @@
-export const Radio = () => {
+"use client";
+
+import clsx from "clsx";
+import React, { memo } from "react";
+
+type RadioProps = {
+  id: string;
+  checked?: boolean;
+  disabled?: boolean;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
@andy-t-wang I think we don't need this prop, it should be controlled by react-hook-form